Tests/LinAlg/test_LinAlg_inner_product.cpp

```diff
@@ -0,0 +1,75 @@
+#include "../test.hpp"
+
+NUMERIC_NSL_TEST_CASE("LinAlg: inner_product 1D", "[LinAlg,inner_product]"){
+    const NSL::size_t size = GENERATE(1,100,200,500,1000);
+    INFO(fmt::format("Size: {}", size));
+
+    // define 2 tensors with ones everywhere
```

Comment:
Define two constant tensors with values 1 and 2?

Reply:
Done with dc0a89f788e459a52b32e891466e5502124a4c0d

Reply:
That fix was applied to the 2D inner product but not the 1D inner product.